### DISCUSSION TITLE
Container Image on Docker Hub/ Quay.io

### DISCUSSION THREAD

**Discussion by galexrt** [LOENS2/pterodactyl_exporter#19] [Ideas]
Could we get the exporter build and pushed to Docker Hub (and preferably Quay.io as an alternative to Docker Hub)?

I can look into helping a bit, e.g., running the container as non-root, in the future if you are interested.

If you have any questions regarding creating a GitHub Actions flow for that or anything else, please let me know.

**Comment by LOENS2** [comment]
The project is already up on Docker Hub: https://hub.docker.com/r/loens2/pterodactyl_exporter . I can put it on Quay.io as well (but regarding the stuff Red Hat has done recently I didn't think about uploading it there). I'm already working on automating CI/CD on my GitLab instance. Running the container non-root would be pretty nice, as I don't see any reason to do so, but unfortunately I'm still pretty bad a docker so I would really appreciate your help.
